### PR TITLE
[wallet] Handle conclude event

### PR DESCRIPTION
**Pull request by cf19drofxots**

### CODE REVIEW

File: packages/wallet/src/contract-tests/test-utils.ts [hunk 1]
```diff
@@ -109,7 +109,7 @@ export async function concludeGame(provider: ethers.providers.JsonRpcProvider, c
     channel,
     allocation: fiveFive,
     destination: [participantA.address, participantB.address],
-    turnNum: 6,
+    turnNum: 7,
```
Comment (snario) [inline packages/wallet/src/contract-tests/test-utils.ts]: Double check that `yarn test:ci:contracts` still passes with this change. I had to handle this in #161

File: packages/wallet/src/redux/sagas/adjudicator-watcher.ts [hunk 2]
```diff
@@ -64,6 +64,9 @@ function* dispatchEventAction(event: AdjudicatorEvent) {
         })
       );
       break;
+    case AdjudicatorEventType.Concluded:
+      yield put(actions.concludedEvent({channelId}));
+      break;
```
Comment (snario) [inline packages/wallet/src/redux/sagas/adjudicator-watcher.ts]: Shouldn't there be a `default:` here?

Comment (cf19drofxots) [reply packages/wallet/src/redux/sagas/adjudicator-watcher.ts]: There should be - what's an appropriate default for this switch block?

File: packages/wallet/src/contract-tests/test-utils.ts (hunk 1 above)
Comment (cf19drofxots) [reply packages/wallet/src/contract-tests/test-utils.ts]: It does

**Pull request merged**